Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -420,7 +420,7 @@ def genS(fragment,howMuch):
 steps['ZTTINPUT']={'INPUT':InputInfo(dataSet='/RelValZTT/%s/GEN-SIM'%(baseDataSetRelease[0],),location='STD')}
 steps['H130GGgluonfusionINPUT']={'INPUT':InputInfo(dataSet='/RelValH130GGgluonfusion/%s/GEN-SIM'%(baseDataSetRelease[0],),location='STD')}
 steps['PhotonJets_Pt_10INPUT']={'INPUT':InputInfo(dataSet='/RelValPhotonJets_Pt_10/%s/GEN-SIM'%(baseDataSetRelease[0],),location='STD')}
-steps['QQH1352TINPUT']={'INPUT':InputInfo(dataSet='/RelValQQH1352T/%s/GEN-SIM'%(baseDataSetRelease[0],),location='STD')}
+steps['QQH1352TINPUT']={'INPUT':InputInfo(dataSet='/RelValQQH1352T_Tauola/%s/GEN-SIM'%(baseDataSetRelease[0],),location='STD')}
```

Comment:
I actually think we should comment this line instead of change it back to the one with string "Tauola", but my opinion is also not strong on that, because of the following analysis:

1.) The string "Tauola" was a historical leftover string in the GEN-SIM dataset input name when you changed the GEN card to a version without "Tauola", where you can see the workflow name has also the string "Tauola" removed: https://github.com/cms-sw/cmssw/commit/5970f82

2.)   This change has also been backported in 71X where the GEN-SIM are being generated: https://github.com/cms-sw/cmssw/commit/98b897a9470b  
    Therefore whenever a new round of GEN-SIM [newer than CMSSW_7_1_0_pre7-PRE_STA71_V3-v1] to be generated, the GEN-SIM inputs can be updated accordingly.  
   
3.) To keep things consistency and clean, I think we should keep this string "Tauola" removed. We just need to comment it to avoid the IB test to do some useless test, in which case, the workflow will automatically re-gen the gen-sim by the IB itself.  

4.) however, on the other hand, this is actually a run1 old workflow, we are not sure we will need it or not in the relval in the future, so, probably we can simply leave with this misleading string "Tauola" inside. In this case, we need also change the 71X one back to be the misleading one with string "Tauola" inside, for consistency. 

5.) but all these are actually some effort for nothing. All what we need is just to suppress the error message, comment this line to bypass the no-more-useful IB input test for this old workflow can be one easier and more consistency solution ....  i think.